### ISSUE TITLE
How to disable all Typescript validation messages?

### ISSUE THREAD

**Issue by chriscalo** [tommasongr/nova-vue#38]
I'm seeing a few Typescript validation messages from Vetur:

![image](https://user-images.githubusercontent.com/867717/123577927-b9fd6f80-d7a2-11eb-8df5-c28cfe0a093f.png)

These aren't useful for me, so I'd like to turn them off.

I tried unchecking all **Validation** options in `nova-vue`'s preferences, but that didn't seem to work:

![image](https://user-images.githubusercontent.com/867717/123577986-dac5c500-d7a2-11eb-9585-71f712ef9c5e.png)

**Comment by chriscalo** [comment]
I'm also seeing this message on occasion, perhaps this is related?

![image](https://user-images.githubusercontent.com/867717/123578254-6c353700-d7a3-11eb-95e9-feae4487d814.png)

**Comment by tommasongr** [comment]
Hi @chriscalo thanks for reporting this.

I just checked and I can confirm that unfortunately almost all the options for Vetur are no longer registered correctly... I'll further investigate. Right now I can't tell if its because of the VLS upgrade I did in v5.0 or for some changes in Nova v7.0... In any case this is going to be a priority issue.

The message from Vue Language Server is not related. It is an automatic warning the Vetur sends. It happens also in VS Code. If Vetur settings work, you will be able to disable them with the setting Ignore Project Warning.
